index.html

```diff
@@ -2127,6 +2134,14 @@ <h2>Capability Delegation</h2>
   <span class="comment">...</span>
 }
         </pre>
+
+In the example above, the DID document for 
+<code>did:example:123456789abcdefghi</code> defines two verificationMethods, 
+<code>#keys-1</code> and <code>#keys-2</code>, that can be used for the 
+purpose of capability delegation. This means that these verification methods 
+can be used to verify delegated capabilities issued by 
+<code>did:example:123456789abcdefghi</code>.
```

Comment:
`delegated capabilities issued by` reads oddly to me. Does this work?
```suggestion
can be used to verify capabilities delegated by
<code>did:example:123456789abcdefghi</code>.
```

Reply:
This works for me. @dlongley?